src/main/webapp/js/instructorCoursesAjax.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function(){
+    var ajaxRequest = function(e) {
+        e.preventDefault();
+        var formData = $(this).serialize();
+        $.ajax({
+            type: 'POST',
+            cache: false,
+            url: $(this).attr('action') + '?' + formData,
+            beforeSend: function() {
+                $('#coursesList').html(
+                    '<img height="75" width="75" class="margin-center-horizontal" src="/images/ajax-preload.gif"/>'
+                );
+            },
+            error: function() {
+                $('#coursesList').html('');
+                setStatusMessage(
+                    'Courses could not be loaded. Click <a href="#" id="retryAjax">here</a> to retry'
+                );
+                $('#retryAjax').click(function(e) {
+                    e.preventDefault();
+                    $('#ajaxForCourses').trigger('submit');
+                });
```

Comment:
Okay, it might not be possible for requests that start immediately after the page loads. Maybe we can think of some way later. For now, this PR is okay to merge.